plugins/SBOMVis/pyproject.toml

```
@@ -0,0 +1,40 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "surfactant-SBOMVis"
+authors = [
+    {name = "Joseph Hand", email = "hand8@llnl.gov"},
+]
+description = "Surfactant SBOM Visualization"
+readme = "README.md"
+requires-python = ">=3.8"
```

```suggestion
requires-python = ">=3.9"
```

Surfactant requires Python >=3.9, so that's a transitive dependency on a newer version of Python for sbomvis.